server/events/vcs/github_client_internal_test.go

```diff
@@ -16,19 +16,20 @@ package vcs
 import (
 	"testing"
 
+	"github.com/runatlantis/atlantis/server/logging"
 	. "github.com/runatlantis/atlantis/testing"
 )
 
 // If the hostname is github.com, should use normal BaseURL.
 func TestNewGithubClient_GithubCom(t *testing.T) {
-	client, err := NewGithubClient("github.com", "user", "pass")
+	client, err := NewGithubClient("github.com", "user", "pass", logging.NewNoopLogger())
```

Comment:
You should be able to pass `nil` for these. SimpleLogger checks if its pointer receiver is nil so it won't panic.

Reply:
Great! I didn't know that